### PR TITLE
Removed unnecessary parentheses in plugins

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Removed unnecessary parentheses in plugins

This PR is part of a set to try to separate some of the changes done in one of my previous batch PR's for plugins, which is still on hold (#12228).
Once the new set is merged it will hopefully reduce the changes in that PR, so it can be reviewed easier and finally be merged.

The changes in this PR are few and very easy to review. In hope that  this will get merged quickly. ;)

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

**Review by andrepereiradasilva — APPROVED** [review]

File: plugins/editors/tinymce/tinymce.php [hunk 1]
```diff
@@ -698,7 +698,7 @@ private function tinyButtons($name, $excluded)
 				// Set some vars
 				$name    = 'button-' . $i . str_replace(" ", "", $button->get('text'));
 				$title   = $button->get('text');
-				$onclick = ($button->get('onclick')) ? $button->get('onclick') : null;
+				$onclick = $button->get('onclick') ? $button->get('onclick') : null;
```
Comment (shur) [inline plugins/editors/tinymce/tinymce.php]: is better to use `?:` comparison operator

File: plugins/system/debug/debug.php [hunk 2]
```diff
@@ -1617,7 +1617,7 @@ protected function displayUntranslatedStrings()
 			if (is_array($occurance) && isset($occurance[0]))
 			{
 				$info = $occurance[0];
-				$file = ($info['file']) ? $info['file'] : '';
+				$file = $info['file'] ? $info['file'] : '';
```
Comment (shur) [inline plugins/system/debug/debug.php]: and here

File: plugins/editors/tinymce/tinymce.php (hunk 1 above)
Comment (frankmayer) [inline plugins/editors/tinymce/tinymce.php]: The ternary replacements will be done in another PR. I would like to keep those small and easy to digest.

File: plugins/system/debug/debug.php (hunk 2 above)
Comment (frankmayer) [inline plugins/system/debug/debug.php]: as above. Thanks for checkoing, though. :)

File: plugins/editors/tinymce/tinymce.php (hunk 1 above)
Comment (shur) [inline plugins/editors/tinymce/tinymce.php]: clear, then it is okay

**Review by shur — APPROVED** [review]
Code review

**Review by ggppdk — APPROVED** [review]

**Comment by ggppdk** [comment]
I have tested this item :white_check_mark: successfully on d4e8830fdbeca599aa4bae3b848058c7bc30ca31<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13212">issues.joomla.org/tracker/joomla-cms/13212</a>.</sub>
